Automattic/wp-calypso
# Jetpack Plans: Add Monthly Plans to Plans List and Plans Compare pages

This adds support for monthly Jetpack plans. When a user has a Jetpack site, they will be able to choose between yearly and monthly jetpack plans, meaning they can pay yearly on each month.

To test;
Apply patch so API endpoints will include new monthly Jetpack plans - D1786-code
Open an account with a Jetpack site connected
Go to plans page for this site i.e. `/plans/{jetpack site name}`
Try the 'Show Monthly Plans' link on the plans page
Try the Compare Plans page while viewing monthly plans and verify that monthly plans are compared
Try the 'Show Yearly Plans' link on plans page
Try a wpcom site to verify that it works as normal
Anything else you can think of...
